src/struct_mv/struct_vector.c

```diff
@@ -1320,3 +1320,69 @@ hypre_StructVectorClone(hypre_StructVector *x)
 
    return y;
 }
+
+/*--------------------------------------------------------------------------
+ *--------------------------------------------------------------------------*/
+
+HYPRE_Int
+hypre_StructVectorElmdivpy( hypre_StructVector *x,
```

Comment:
I didn't see this one in ParVector.  What does it do?

Reply:
`Elmdivpy` was added in the past for `hypre_ParVector`. It computes `y = y + x ./ b`

https://github.com/hypre-space/hypre/blob/bc07da4bbfa55e4ff2c49e9312c5705d463e0772/src/parcsr_mv/par_vector.c#L688-L704